bin/plugin/commands/performance.js

```diff
@@ -260,6 +262,33 @@ async function runPerformanceTests( branches, options ) {
 	);
 
 	log( '>> Starting the WordPress environment' );
+	if ( options.wpVersion ) {
+		// In order to match the topology of ZIP files at wp.org, remap .0
+		// patch versions to major versions:
+		//
+		//     5.7   -> 5.7   (unchanged)
+		//     5.7.0 -> 5.7   (changed)
+		//     5.7.2 -> 5.7.2 (unchanged)
+		const zipVersion = options.wpVersion.replace( /^(\d+\.\d+).0/, '$1' );
```

Comment:
IMO, we could drop this altogether and make it the caller's concern to choose the right version format (mostly thinking YAGNI here, since our only use case is guaranteed to use the correct format). Not a big deal tho, I don't mind having it here.

Reply:
True, though in this case I feel it would open the door for mistakes, because someone operating the `--wp-version` flag doesn't need to know that the flag results in a specific ZIP file being pulled later by wp-env. `--wp-version 5.7.0` should be an accepted value, so let's ensure it doesn't break.